### ISSUE TITLE
Control panel changes

### ISSUE THREAD

**Issue by volga629-1** [VoIP-SAAS/opensips-cp#3]
1. Move Show Entries to left botom
2. Action dropdown and move all tool bar tools to it 
     a. Add column 
     b. Show/Hide
     c. Sort
     d. Column Filter ( add/remove filter icon from selected column )
     e. Edit
     f. Delete
3. Search Bar move beside the action drop down.

**Comment by volga629-1** [comment]
![OpenSIPS Control Panel.png](https://images.zenhubusercontent.com/5dfbbe66f56c22b686cf99fe/395fa75f-3e33-47d1-a67b-64d089f9c36a)

**Comment by volga629-1** [comment]
By default  filter icon  under columns will be not visible.  Filter Actions  tool we will have selector ( list ) of all columns.
Filter icon will show up under selected columns from filter action tool.

**Comment by volga629-1** [comment]
Actions look and fill 

![Actions-Drop-Down.png](https://images.zenhubusercontent.com/5dfbbe66f56c22b686cf99fe/48b41415-3b41-4842-a531-28bbc7728d9a)

**Comment by blugee** [comment]
@volga629-1  I don't think so, we need a selector for this one. We can just simply set enable action menu by default.
